easybuild/easyblocks/v/vmd.py

```diff
@@ -150,6 +165,40 @@ def configure_step(self):
         # change to 'src' subdirectory, ready for building
         change_dir(os.path.join(vmddir, 'src'))
 
+    def build_step(self):
+        """Custom build step for VMD."""
+        super(EB_VMD, self).build_step()
+
+        vmddir = os.path.join(self.cfg['start_dir'], '%s-%s' % (self.name.lower(), self.version))
+        self.have_stride = False
+        # Build Surf, which is part of VMD as of VMD version 1.9.3
+        if LooseVersion(self.version) >= LooseVersion("1.9.3"):
+            change_dir(os.path.join(vmddir, 'lib', 'surf'))
+            surf_cmd = 'make CC="%s" OPT="%s"' % (os.environ['CC'], os.environ['CFLAGS'])
+            run_cmd(surf_cmd)
+            # Build Stride if it was downloaded
+            change_dir(os.path.join(vmddir, 'lib', 'stride'))
```

Comment:
What if this directory doesn't exist?

Reply:
Ahhh, need to deal with that. best approach being??

Reply:
`os.path.exists` before `change_dir` + proper error reporting

Reply:
since stride isn't mandatory, should we just issue a warning or raise an error do you think?